examples/dt/uni02beta/control-plane/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../dt/uni02beta
+
+resources:
+  - nncp/values.yaml
+  - service-values.yaml
+  - cinder-netapp-secrets.yaml
+  - manila-netapp-secrets.yaml
```

Comment:
Since these are being handled in `dt/uni02beta/kustomization.yaml`, I would suggest moving them to that dir.